.travis.yml

```diff
@@ -1,5 +1,8 @@
 language: node_js
 node_js:
-  - "5.4"
+  - "8"
 script:
   - npm test
+before_install:
```

Comment:
Seems like yarn is supported out of the box https://docs.travis-ci.com/user/languages/javascript-with-nodejs#Travis-CI-supports-yarn so not sure this is necessary

Reply:
I got issues before running Node 8 that were due to the pre-installed version being 1.3.2 - this was actually resolved by the Node 8 update but I had by then already added this part that updates yarn to be always the latest version. There is some discussion on the subject here: https://github.com/travis-ci/travis-ci/issues/7566

Reply:
Let's remove this if unnecessary